sdks/java/core/src/main/java/org/apache/beam/sdk/testing/RunnableOnService.java

```diff
@@ -31,4 +31,4 @@
  *     }
  * </code></pre>
  */
-public interface RunnableOnService {}
+public interface RunnableOnService extends NeedsRunner {}
```

Comment:
Nice move. We might want to deprecate `RunnableOnService` now. There is a separate category "actually important to establish the capability matrix" which we might call `RunnerCapabilityTest extends NeedsRunner` but there are certainly far fewer of these than `RunnableOnService` tests.